.github/workflows/dev-efiling-reviewer-api-build.yaml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [master]
   workflow_dispatch:
-    branches: [master]
```

Comment:
if you're removing this, remove the event too please

Reply:
ahahaha

Reply:
I was pushing from my branch